src/main/js/common/MoneroConnectionManager.js

```diff
@@ -394,7 +395,7 @@ class MoneroConnectionManager {
   /**
    * Set the maximum request time before its connection is considered offline.
    * 
-   * @param {int} timeoutInMs - the timeout before the connection is considered offline
+   * @param {integer} timeoutInMs - the timeout before the connection is considered offline
```

Comment:
If I'm not mistaken, {number} would be more standard.

Reply:
More standard AND less completely incorrect! Yes, this entire PR is a bit fubar, and I'm about to revoke it.